src/Helpers/AsyncPipe.test.ts

```diff
@@ -0,0 +1,18 @@
+import { describe } from 'riteway'
+import { asyncPipe } from './asyncPipe'
+
+describe('asyncPipe', async (should: any) => {
+  const { assert } = should('')
+
+  {
+    const inc = (number = 0) => Promise.resolve(number + 1)
+    const actual = await asyncPipe(inc)(0)
```

Comment:
I think it would be good to have a test with 3 or more functions in the pipe to make sure data flows correctly. Here are a couple of function ideas, double, and add10.

Reply:
@kennylavender I'd go with merging your PR asap and using Ramda's pipeP as you commented instead, and avoid investing time and effort into testing asyncPipe. What do you think?

Reply:
@lautarodragan yup I agree :D